lib/actor.js

```diff
@@ -195,7 +195,8 @@ class Actor {
       name: this.name,
       children: new Map(this.childReferences),
       sender,
-      log: this.log
+      log: this.log,
+      mailbox: this.mailbox.toArray()
```

Comment:
Should this not be in the supervision context rather?

Reply:
Moved it out to an intermediate type between SupervisionContext and ActorContext to be shared with afterStop, as discussed.